### DISCUSSION TITLE
Converting full-colour GIF to 256-colour

### DISCUSSION THREAD

**Discussion by bbergeron0** [ImageMagick/ImageMagick#5962] [Help]
### ImageMagick version

7.1.0-44

### Operating system, version and so on

Gentoo

### Description

Hi, I need to convert full-colour GIFS whith sub-images to old school, 256-colour, single-block GIFS (Here's what I'm talking about: https://en.wikipedia.org/wiki/GIF#Palettes).
The output should not have any sub-images/sub-frames. Can ImageMagick do that?

**Comment by snibgo** [comment]
I don't know the answer. Have you tried this? If you don't have IM, you can post sample inputs and we can try this.

**Comment by fmw42** [comment]
I do not know if Imagemagick even supports full color GIFs.  I have never heard that use in Imagemagick.  Perhaps one of the developers can comment.
